bin/citrea/tests/e2e/sequencer_behaviour.rs

```diff
@@ -178,38 +178,40 @@ async fn test_sequencer_commitment_threshold() {
 
     seq_test_client.send_publish_batch_request().await;
 
-    for _ in 0..10 {
-        for _ in 0..100 {
+    for i in 1..11 {
+        for _ in 0..300 {
             let address = Address::random();
             let _pending = seq_test_client
                 .send_eth(address, None, None, None, 1u128)
                 .await
                 .unwrap();
         }
         seq_test_client.send_publish_batch_request().await;
+        wait_for_l2_block(&seq_test_client, i, Some(Duration::from_secs(10))).await;
     }
 
     wait_for_l2_block(&seq_test_client, 11, Some(Duration::from_secs(60))).await;
 
-    // At block 725, the state diff should be large enough to trigger a commitment.
+    // After block 9/10, the state diff should be large enough to trigger a commitment.
```

Comment:
don't be surprised by the huge height changes in the comments, they were wrong in the first place. this test shows how the state diff improved by the tx counts below